Nos/Views/ProfileView.swift

```diff
@@ -23,6 +27,25 @@ struct ProfileView: View {
         _events = FetchRequest(fetchRequest: author.allPostsRequest())
     }
     
+    func refreshProfileFeed() {
+        // Close out stale requests
+        if !subscriptionIds.isEmpty {
+            relayService.sendCloseToAll(subscriptions: subscriptionIds)
+            subscriptionIds.removeAll()
+        }
+
+        if let currentUserPublicKey = CurrentUser.author.hexadecimalPublicKey {
+            let authors = [currentUserPublicKey]
+            let textFilter = Filter(authorKeys: authors, kinds: [.text], limit: 100)
+            let textSub = relayService.requestEventsFromAll(filter: textFilter)
+            subscriptionIds.append(textSub)
+            
+            let metaFilter = Filter(authorKeys: authors, kinds: [.metaData, .contactList], limit: 10)
```

Comment:
Can you split this into two separate filters: one for metadata and one for .contact list. Both can have limit 1. This fixes the bug where if the last 10 events are .contactList we may never get any metadata.

Reply:
If we want to always have only one kind per filter, then we should just update Filter to accept a single EventKind instead of an array of them. What do you think?

Reply:
@mplorentz I've made the update on this.

Reply:
@cjorgensen yeah that's a good point. We can make that change in a future PR whenever it's convenient.